raycicmd/dep_graph_test.go

```diff
@@ -0,0 +1,134 @@
+package raycicmd
+
+import (
+	"testing"
+
+	"reflect"
+	"sort"
+)
+
+type testDepNode struct {
+	d []string
+	m bool
+}
+
+func newTestDepNode(deps []string) *testDepNode {
+	return &testDepNode{d: deps}
+}
+func (n *testDepNode) deps() []string { return n.d }
+func (n *testDepNode) mark()          { n.m = true }
+
+type testDepGraph struct{ g map[string]*testDepNode }
+
+func newTestDepGraph(g map[string][]string) *testDepGraph {
+	nodes := make(map[string]*testDepNode)
+	for k, deps := range g {
```

Comment:
can we use words for some of these single character; i find myself referring back to the struct definition to understand the meaning for some of these (e.g. g --> graph, k --> id/nodeId?)

Reply:
done.